src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -2168,23 +2217,42 @@ public static boolean areAnyGencodeFuncotation(final List<Funcotation> funcotati
         Utils.nonNull(metadata);
```

Comment:
Yes, and I changed the test to test `createFuncotations`, rather than `createFuncotationsFromMetadata`, which is a private method called by `createFuncotations`.  These two methods basically do the same thing, but `createFuncotations` has the error checking in it.

Done.